drivers/scheduler/k8s/specs/mysql/px-mysql-snapshots.yaml

```diff
@@ -1,7 +0,0 @@
-##### Portworx Stork Snapshot
-apiVersion: volumesnapshot.external-storage.k8s.io/v1
-kind: VolumeSnapshot
-metadata:
```

Comment:
why did we remove this file?

Reply:
I'll bring it back once my new PR for spec management comes into play.